frontend/src/components/user-management/AdminTab.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import {
+  Text,
+  VStack,
+  Input,
+  InputGroup,
+  InputRightElement,
+  HStack,
+} from "@chakra-ui/react";
+
+import { User } from "../../types/UserTypes";
+
+import { SearchOutlineIcon } from "../../assets/icons";
+
+type UserProperty = "firstName" | "email" | "school";
```

Comment:
```suggestion
```
pay attention when you're fixing merge conflicts! check this file and remove all code and imports that aren't used